src/main/java/io/javarig/generator/list/ListGenerator.java

```diff
@@ -13,6 +13,11 @@
 import java.lang.reflect.Type;
 import java.util.List;
 
+/**
+ * a type generator that generates a list instance,
+ * <p>
+ * note: generates an ArrayList if not given otherwise
```

Comment:
This information is not handled here. The code doesn't generate ArrayList if no implementation type was given. We did that in the TypeEnum when we gave the LIST enum ArrayListGenerator.
I find this command is confusing.

Reply:
check **Nonlocal Information** in chapter 4 of **Clean Code** book
https://thixalongmy.haugiang.gov.vn/media/1175/clean_code.pdf